### PR TITLE
Parity handling

### PR DESCRIPTION
**Pull request by antoine-de**
handle parity in realtime

The parity is the fact that several trips can be impacted with one real time message

linked to http://jira.canaltp.fr/browse/HOR-100

### CODE REVIEW

**Comment by antoine-de** [comment]
Jenkins, test this please

File: tests/integration/ire_test.py [hunk 1]
```diff
@@ -353,21 +372,29 @@ def test_ire_two_trip_removal_post_twice(mock_rabbitmq):
...
-def test_ire_trip_removal_parity(mock_rabbitmq):
+def test_ire_trip_with_parity(mock_rabbitmq):
     """
-    simple parity trip removal post
+    a trip with a parity has been impacted, there should be 2 VJ impacted
     """
     ire_6113 = get_ire_data('train_6113_trip_removal.xml')
     ire_6113_14 = ire_6113.replace('<NumeroTrain>006113</NumeroTrain>',
-                                   '<NumeroTrain>006113/14</NumeroTrain>')
+                                   '<NumeroTrain>006113/4</NumeroTrain>')
```
Comment (pbougue) [inline tests/integration/ire_test.py]: @prhod : this could have stayed the same (6113/14), I guess.

Comment (antoine-de) [inline tests/integration/ire_test.py]: yes it's exactly the same

File: kirin/ire/model_maker.py [hunk 2]
```diff
@@ -136,37 +157,42 @@ def build(self, rt_update):
...
-
-        return vjs
+        train_numbers = headsigns(get_value(xml_train, 'NumeroTrain'))
+
+        vjs = {}
+
+        for train_number in train_numbers:
+            # to get the date of the vj we use the start/end of the vj + some tolerance
+            # since the ire data and navitia data might not be synchronized
+            vj_start = as_date(get_value(xml_train, 'OrigineTheoriqueTrain/DateHeureDepart'))
```
Comment (patochectp) [inline kirin/ire/model_maker.py]: To be placed outside the loop

**Comment by antoine-de** [comment]
@kinnou02 can you have a look at this ?

**Pull request merged**